src/langs/de.js

```diff
@@ -0,0 +1,17 @@
+//Add strings
+gdprCookieNoticeLocales.en = {
```

Comment:
This should be gdprCookieNoticeLocales.de

Then in the gulpfile all languages should be loaded, so you can access them.
Currently only en is loaded.

Reply:
> This should be gdprCookieNoticeLocales.de

Correct. Fixed in #34.